index.html

```diff
@@ -2,12 +2,38 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
     <!-- don't forget to link your styles -->
   </head>
   <body>
+  <header>
+    <div>
+      <img class="logo" src="images/spotify-logo.png" alt="Spotify logo"  />
+    <nav class="nav-bar">
+      <ul class="nav-bar-ul">
+        <li> <a class="nav-bar-item" href="#premium"> Premium </a> </li>
+        <li> <a class="nav-bar-item" href="#discover"> Discover </a> </li>
+        <li> <a class="nav-bar-item" href="#help"> Help </a> </li>
+        <li> <a class="nav-bar-item" href="#download"> Download </a> </li>
+      
+      </ul>
+    </nav>
+    </div>
+  </header>
+
+
+    <section class="sectionOne">
+      <div>
+        <h1> Music for everyone. </h1>
+        <br>
+        <h3 class= "free">Spotify is now free on mobile, tablet and computer. <br> 
+          Listen to the right music, wherever you are.</h3>
+      </div>
+        </section>>
```

Comment:
Looks like there is an extra ```>``` closing tag that snuck along here 🤭. 
You can remove it 😄 
```suggestion
        </section>
```